packages/graphql/tests/integration/utils/tests-helper.ts

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Cypher from "@neo4j/cypher-builder";
+import type { ExecutionResult, GraphQLArgs } from "graphql";
+import { graphql as graphqlRuntime } from "graphql";
+import * as neo4j from "neo4j-driver";
+import { Memoize } from "typescript-memoize";
+import type { Neo4jGraphQLConstructor, Neo4jGraphQLContext } from "../../../src";
+import { Neo4jGraphQL } from "../../../src";
+import { createBearerToken } from "../../utils/create-bearer-token";
+import { UniqueType } from "../../utils/graphql-types";
+
+const INT_TEST_DB_NAME = "neo4jgraphqlinttestdatabase";
+const DEFAULT_DB = "neo4j";
+
+let helperLock = false;
+
+export class TestHelper {
+    private database: string = DEFAULT_DB;
```

Comment:
I see below that we verify the connectivity using the variable `USE_DEFAULT_DB` IMO it would make sense that `checkConnectivity` and other methods target the same database, thoughts?

Reply:
mmm, in theory, checkConnectivity will update this value if needed, I'm not sure what you mean

Reply:
I think that the `env.USE_DEFAULT_DB` is used only to verify the connectivity but is ignored for other purposes like `runCypher` or `runGraphQL`, I don't have tested tho, though it's just my impression reading the code!